devtools/create_project/xcode.cpp

```diff
@@ -588,6 +600,9 @@ void XcodeProvider::setupFrameworksBuildPhase(const BuildSetup &setup) {
 	if (CONTAINS_DEFINE(setup.defines, "USE_ZLIB")) {
 		frameworks_iOS.push_back("libz.tbd");
 	}
+	if (CONTAINS_DEFINE(setup.defines, "USE_DISCORD")) {
+		frameworks_iOS.push_back("libdiscord-rpc.a");
```

Comment:
I don't think it makes sense to use the discord feature with iOS. The way it is implemented currently means it is tied to the SDL backends. Furthermore I think it requires Discord and ScummVM running at the same time, which is not possible on iOS.

Reply:
Correct, discord-rpc only does anything on desktop platforms. There's no game activity on mobile Discord.